flink-streaming-java/src/main/java/org/apache/flink/streaming/util/FieldAccessor.java

```diff
@@ -182,11 +182,14 @@ public R set(R record, F fieldValue) {
 			@SuppressWarnings("unchecked")
 			CompositeType<R> cType = (CompositeType<R>) type;
 
+			if(field.contains(".")) {
+				throw new IllegalArgumentException("The Pojo field accessor currently doesn't support nested POJOs");
```

Comment:
Thank you for reviewing my pull request. I've opened FLINK-3702 for that.

See also here: https://github.com/apache/flink/pull/1851/files#diff-e5e091a2c1e4bf850ef93e2010fe4c81R132